.github/workflows/canary-integration-test.yml

```diff
@@ -55,7 +55,7 @@ jobs:
         cd cluster/examples/kubernetes/ceph/
         kubectl create -f crds.yaml -f common.yaml
         # skipping folders and some yamls that are only for openshift.
-        kubectl create  $(ls -I scc.yaml -I operator-openshift.yaml -I "*.sh" -I "*.py" -p | grep -v / | awk ' { print " -f " $1 } ') --dry-run
+        kubectl create  $(ls -I scc.yaml -I operator-openshift.yaml -I object-openshift.yaml -I "*.sh" -I "*.py" -p | grep -v / | awk ' { print " -f " $1 } ') --dry-run
```

Comment:
how about skipping if a filename contains openshift in it? if we do so we don't need to keep on updating this check?

Reply:
Done.